File: backend/FluffitMember/src/main/java/com/ssafy/fluffitmember/member/dto/Response/RankDto.java
package com.ssafy.fluffitmember.member.dto.Response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class RankDto {
    private int rank;
    private String nickname;
    private String flupetNickname;
    private String flupetImageUrl;
}

File: backend/FluffitFlupet/src/main/kotlin/com/ssafy/fluffitflupet/dto/RankDto.kt
package com.ssafy.fluffitflupet.dto

data class RankDto(
    var rank: Int,
    var userNickname: String = "",
    var userId: String,
    var lifetime: Long,
    var flupetNickname: String,
    var imageUrl: List<String>
)
